index.html

```diff
@@ -1,32 +1,164 @@
+                 <ul class="language-container">
+                   <li class="language html">HTML</li>
+                   <li class="language css">CSS</li>
+                   <li class="language java">Javascript</li>
+                 </ul>
+                 <button class="card-btn">See project</button>
+           </div>
+    </section>
+    <section class="contact-container">
+        <div class="one">
+         <contact class="about"><img src="/img/Primary Text.svg" alt="" class="about-pic"></contact>
+         <p class="bio">Hello I’m a software developer!
+             I can help you build a product , feature or website. 
+             Look through some of my work and experience! 
+             If you like what you see and have a project you need coded, don’t hesitate 
+             to contact me.
+         </p>
+         <ul> 
+            <li><a href="#"><img src="/img/LET’S CONNECT.svg" alt="" class="contact bottom"></a></li>
+           </ul>
+           <ul class="icon bicon">
+             <li class="github"><a href="#"><img src="/img/github.svg" alt="github"></a></li>   
+             <li class="linkdin"><a href="#"><img src="/img/Linkedin icon.svg" alt="linkedin"></a></li>
+             <li class="v"><a href="#"><img src="/img/v.svg" alt="v"></a></li>
+             <li class="twiter"><a href="#"><img src="/img/twiter.svg" alt="twiter"></a></li>
+             <li class="m"><a href="#"><img src="/img/m.svg" alt="m"></a></li>
+             
+             
+         </ul>
+         <button class="card-btn resume">Get my resume</button>
+        </div>
+        <div class="two">
+            <div class="title-language">
+                <h2 class="tl">Language</h2>
+                <img src="/img/Icon - Chevron - Down - Larger.svg" alt="" class="dropdow">
+            </div>
+            <ul class="fotter-java">
+              <li class="space"><img src="/img/java.svg" alt="Javascript" class="language-pic"></li>
+              <li class="space"><h3 class="lp-title">Javascript</h3></li>
+            </ul>
+            <ul class="fotter-html">
+              <li class="space"><img src="/img/html.svg" alt="html" class="language-pic"></li>
+              <li class="space"><h3 class="lp-title">HTML</h3></li>
+            </ul>
+            <ul class="fotter-css">
+              <li class="space"><img src="/img/css.svg" alt="css" class="language-pic"></li>
+              <li class="space"><h3 class="lp-title">CSS</h3></li>
+            </ul>
+            <ul class="frameworks-container">
+              <li class="framework">Frameworks</li>
+              <li class="right-arow"><img src="/img/Disabled.svg" alt="right-arow" class="arow"></li>
+            </ul>
+            <ul class="skills-container">
+              <li class="skills">Skills</li>
+              <li class="right-arow"><img src="/img/Disabled.svg" alt="right-arow" class="arow"></li>
+            </ul>
```

Comment:
- [ ] Please make sure that the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`. And the technology in each category(e.g., "JavaScript", "HTML" and "CSS") are also a list `ul` and each of them a `li` as given in the design guidelines

  > _Please use ul and li for categories in "Work" section. Html, CSS, and JavaScript should be list items._
  > _Please use ul and li for categories in "About myself" section. Languages, Frameworks, and Skills should be list items._

- [ ] Also, kindly implement the following for getting a design close to the Figma template
- Increase the space between the button and the list
- Add a background colroe as given in the Figma template for `html`, `css` and `javascript`. Also, don't forget to apply the correct space/padding/margin and border radius for it.
- Reduce the space between image and the text, `html`, `css` and `javascript`
- Align all the arrows as given in the guidelines
- Add a divider for frameworks and skills

Your design | Given design
---| ---|
![image](https://github.com/ali-techworld/Portfolio-Setup/assets/106119105/7bcccda3-8430-4420-9d78-628e7a20cd45) | ![image](https://github.com/ali-techworld/Portfolio-Setup/assets/106119105/0aabae2c-6bad-46ed-b6ae-47f56897cbc4)